app/code/Magento/InventoryCatalog/Plugin/CatalogInventory/Model/ResourceModel/Stock/Status/AdaptAddIsInStockFilterToCollectionPlugin.php

```diff
@@ -11,9 +11,10 @@
 use Magento\CatalogInventory\Model\ResourceModel\Stock\Status;
 use Magento\InventoryCatalog\Model\GetStockIdForCurrentWebsite;
 use Magento\InventoryCatalog\Model\ResourceModel\AddIsInStockFilterToCollection;
+use Magento\InventoryCatalogApi\Api\DefaultStockProviderInterface;
 
 /**
- * Adapt adding is in stock filter to collection for multi stocks.
+ * Class AdaptAddIsInStockFilterToCollectionPlugin
```

Comment:
```suggestion
 * Adapt adding is in stock filter to collection for multi stocks.
```

It is the best practice to keep comments descriptive and meaningful. Previous comment seemed to provide more context on this class.

Reply:
As per new docblock, every function should have a short description that's why I moved this comment [here](https://github.com/magento/inventory/blob/dcc19504b5d1548e94ae283fb62868619c23e457/app/code/Magento/InventoryCatalog/Plugin/CatalogInventory/Model/ResourceModel/Stock/Status/AdaptAddIsInStockFilterToCollectionPlugin.php#L47) just above the function

Reply:
Same applies to classes, not only methods. It's ok to make class and method doc block similar, but avoiding meaningless comments